lib/shared/require-or-import.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var pathToFileURL = require('url').pathToFileURL;
+
+var importESM;
+try {
+  importESM = new Function('id', 'return import(id);');
+} catch (e) {
+  importESM = null;
+}
+
+function requireOrImport(path, callback) {
+  var err = null;
+  var cjs;
+  try {
+    cjs = require(path);
+  } catch (e) {
+    if (pathToFileURL && importESM && e.code === 'ERR_REQUIRE_ESM') {
+      var url = pathToFileURL(path);
```

Comment:
I just added this as I realized that `import()` fails if providing a file path on Windows.